src/main.tsx

```diff
@@ -1,10 +1,39 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
+import Home from './Home.tsx'
 import './index.css'
+import { RouterProvider, createBrowserRouter } from 'react-router-dom'
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Home />
+  },
+  {
+    path: "/:id",
+    element: <Home />
+  },
+  {
+    path: "/onboarding",
+    element: <></>
+  },
+  {
+    path:"/dashboard",
+    element: <h1>Dashboard goes here</h1>
+  },
+  {
+    path: "/event/:eventId", //passes a params object to element containing :id
```

Comment:
instead of using eventId, I was thinking of using the event name instead like /event/pmc-product-challenge for readability. let's make the path `/event/:blob`